ObjectStreaming/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you please all these extra files into .gitignore?

Reply:
sure sir